infrastructure-common/src/main/kotlin/dsl/Predicates.kt

```diff
@@ -1,7 +1,5 @@
 package com.redmadrobot.build.dsl
 
-import org.gradle.api.Project
-
 /** Returns `true` if build is running on CI. */
-public val Project.isRunningOnCi: Boolean
-    get() = providers.environmentVariable("CI").forUseAtConfigurationTime().orNull == "true"
+public val isRunningOnCi: Boolean
+    get() = System.getenv("CI") == "true"
```

Comment:
What is the reason for the return this change?

Reply:
[This](https://docs.gradle.org/7.4/userguide/upgrading_version_7.html#for_use_at_configuration_time_deprecation) one 🙃